CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Task] Display age and estimated duration [#4083](https://xcp-ng.org/forum/topic/4083/task-list-enhancement) (PR [#5530](https://github.com/vatesfr/xen-orchestra/pull/5530))
```

Comment:
In the changelog, `#4083` refers to an GitHub issue (or PR), which it's not, it's confusing, please find a better identifier :slightly_smiling_face:

Reply:
Can i remove the identifier and the link of the forum in the changelog or they are needed ?

Reply:
IMHO, you can remove them, as long as it's still in the commit/PR.